client/android/build.gradle

```diff
@@ -22,6 +22,8 @@ buildscript {
     }
 
     dependencies {
+    	classpath 'com.google.gms:google-services:4.3.2'
+    	classpath 'com.google.firebase:firebase-crashlytics-gradle:2.1.1' 
```

Comment:
Maybe we should uncomment Firebase init, which was turned off in previos commit?

Reply:
Crashlytics is a good thing, but I'm not sure that collecting of some logs is fit into our ideology. We can register new [google-services.json](https://github.com/amnezia-vpn/amnezia-client/pull/187/files#diff-06a86e87a0d834bd8d75c40dcadcbfb80c6415cd3faca61fd4db82d0c0e3bb37) into corporete acc and turn it on.